modules/compute/gke-node-pool/main.tf

```diff
@@ -44,6 +44,7 @@ resource "google_container_node_pool" "node_pool" {
   name           = var.name == null ? var.machine_type : var.name
   cluster        = var.cluster_id
   node_locations = var.zones
+  version        = var.is_gke_sandbox ? var.node_version : null
```

Comment:
We should not add `is_gke_sandbox` here, can we not expose the version in all cases ?